zcbor/zcbor.py

```diff
@@ -2269,12 +2269,12 @@ def range_checks(self, access):
 
     # Return the full code needed to encode/decode this element, including children,
     # key and cbor, excluding repetitions.
-    def repeated_xcode(self, union_uint=None):
+    def repeated_xcode(self, union_int=None):
         range_checks = self.range_checks(self.val_access())
         xcoder = {
             "INT": self.xcode_single_func_prim,
-            "UINT": lambda: self.xcode_single_func_prim(union_uint),
-            "NINT": self.xcode_single_func_prim,
+            "UINT": lambda: self.xcode_single_func_prim(union_int),
```

Comment:
I don't get it, is `union_int` an int or a `UINT` ?

Reply:
From what i read it's a NINT which is int in Cland